gno.land/cmd/gnoland/genesis_txs.go

```diff
@@ -37,3 +40,29 @@ func newTxsCmd(io commands.IO) *commands.Command {
 func (c *txsCfg) RegisterFlags(fs *flag.FlagSet) {
 	c.commonCfg.RegisterFlags(fs)
 }
+
+// appendGenesisTxs saves the given transactions to the genesis doc
+func appendGenesisTxs(genesis *types.GenesisDoc, txs []std.Tx) error {
+	// Initialize the app state if it's not present
+	if genesis.AppState == nil {
+		genesis.AppState = gnoland.GnoGenesisState{}
+	}
+
+	state := genesis.AppState.(gnoland.GnoGenesisState)
```

Comment:
shall we check if AppState can be cast to GnoGenesisState and if not, return an error? just to avoid an unhandled panic here.

Reply:
Good catch:

[a04242f](https://github.com/gnolang/gno/pull/2327/commits/a04242fa867c63e54db2a24f03f6c52852b67a40)